R/group_apply.R

```diff
@@ -0,0 +1,119 @@
+#' Apply a function over grouping(s) of data
+#'
+#' A function to apply `palaeoverse` functionality across subsets (groups) of
+#' data, delineated using one or more variables. `data.frame` related functions
+#' (e.g. `nrow`, `ncol`, `lengths`, `unique`) may also be used.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences.
+#' This dataframe must contain the grouping variables and the necessary
+#' variables for the function you wish to call (see function specific
+#' documentation for required columns).
+#' @param group \code{character}. A vector of column names, specifying the
+#' grouping (e.g. "collection_no", "stage_bin").
+#' @param fun \code{function}. The function you wish to apply to
+#' `occdf`. See details for compatible functions.
+#' @param ... Additional arguments available in the called
```

Comment:
```suggestion
#' @param ... Additional arguments required by, or available to, the called
```
I think you could say this and delete the second sentence (maybe add 'see examples' to the end).

Reply:
Hmmm I think it's better to spell this out clearly here, particularly for overwriting default arguments.

Reply:
Sure, whatever you think is most helpful!